The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/3866203636/jobs/6590173592) [(plain)](https://github.com/rust-lang/rust/commit/00ed01d684b5f58acd36449a1e2e91d21d49f6cd/checks/10507135272/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
configure: rust.debug-assertions := True
configure: rust.overflow-checks := True
configure: llvm.assertions      := True
configure: dist.missing-tools   := True
configure: build.configure-args := ['--enable-sccache', '--disable-manage-submodu ...
configure: writing `config.toml` in current directory
configure: 
configure: run `python /checkout/x.py --help`
Attempting with retry: make prepare
---
    --> compiler/rustc_trait_selection/src/traits/error_reporting/suggestions.rs:214:6
     |
214  |     );
     |      ^
     = note: expected fn pointer `fn(&TypeErrCtxt<'_, 'tcx>, &rustc_infer::traits::Obligation<'tcx, rustc_middle::ty::Predicate<'tcx>>, &mut Diagnostic, Binder<'_, _>)`
                found fn pointer `fn(&TypeErrCtxt<'_, 'tcx>, &rustc_infer::traits::Obligation<'tcx, rustc_middle::ty::Predicate<'tcx>>, &mut Diagnostic, Binder<'_, _>) -> bool`
error[E0308]: mismatched types
   --> compiler/rustc_trait_selection/src/traits/error_reporting/mod.rs:876:28
    |
    |
876 |                         if self.suggest_add_clone_to_arg(&obligation, &mut err, trait_predicate) {
    |                            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected `bool`, found `()`
Some errors have detailed explanations: E0053, E0308.
For more information about an error, try `rustc --explain E0053`.
error: could not compile `rustc_trait_selection` due to 2 previous errors
warning: build failed, waiting for other jobs to finish...
```

</details>

--- a/src/test/ui/suggestions/issue-106443-sugg-clone-for-bound.stderr
+++ b/src/test/ui/suggestions/issue-106443-sugg-clone-for-bound.stderr
@@ -6,7 +6,7 @@ LL |     foo(s);
    |
 help: consider further restricting this bound
    |
-LL | fn bar<T: X + Clone>(s: &T)  {
+LL | fn bar<T: X + Clone>(s: &T) {
    |             +++++++
 help: consider using clone here
    |

